src/round.rs

```diff
@@ -0,0 +1,22 @@
+pub trait Round: Copy {}
+
+macro_rules! impl_round {
+    ($t:ty) => {
+        impl Round for $t {}
+        impl Round for ($t, $t) {}
+        impl Round for ($t, $t, $t) {}
+        impl Round for ($t, $t, $t, $t) {}
```

Comment:
What are these for?

Reply:
This is a convenience implementation of the round abstraction. If we are under pbft, the rules of tendermint wouldn't specify the ruleset of this crate, and this allows us to implement tendermint natively.

Reply:
I'm just not sure why we need more than (round) or (block height, round)? This genericism may make it unnecessarily complex to implement traits like partialeq etc.

Reply:
+1 on we don't need things to be too generic. It's just Tendermint.